lib/src/base-styles/_nx-globals.scss

```diff
@@ -18,19 +18,45 @@ $-nx-scrollable-default-height: 400px;
   color: var(--nx-color-link);
   cursor: pointer;
 
+  .nx-code {
+    color: var(--nx-color-link);
+  }
+
   &:hover {
     color: var(--nx-color-link-hover);
+
+    .nx-code {
+      color: var(--nx-color-link-hover);
+    }
   }
 
   &:active {
     color: var(--nx-color-link-active);
+
+    .nx-code {
+      color: var(--nx-color-link-active);
+    }
   }
 
   &:focus {
-    border-radius: var(--nx-border-radius);
+    // outline-offset scales the border-radius so this
+    // was changed to 1px to get it visually close to
+    // var(--nx-border-radius) or (6px).
+    border-radius: 1px;
     color: var(--nx-color-link-focus);
     outline: 1px solid var(--nx-color-interactive-border-focus);
-    outline-offset: 2px;
+    outline-offset: 4px;
+
+    .nx-code {
+      color: var(--nx-color-link-focus);
+    }
+  }
+
+  .nx-icon {
```

Comment:
Should this apply to any `.nx-icon` within a link or just the external link icon?

Reply:
> Should this apply to any .nx-icon within a link or just the external link icon?

Thanks that's a good point: Updated it to target that icon specifically here:
d887eb2500280aec071ede3466f809ce6a022191